src/federation.rs

```diff
@@ -364,15 +512,39 @@ mod tests {
     }
 
     fn valid_federation() -> Federation {
+        let sigma = ECScalar::from(
+            &BigInt::from_str_radix(
+                "16b63d6f3b5a88762d4477b843b857a3bf86677c7044db22a9aada2eb17d0641",
+                16,
+            )
+            .expect("Failed to parse BigInt"),
+        );
+
+        let v_bytes =
+            Vec::from_hex("dde06d981f17045b11c8db7b47846ceca4825f286756440ea158e0b2dba86028")
+                .expect("Failed to decode hex string");
+        let v = GE::from_bytes(&v_bytes).expect("Failed to create GE from bytes");
+
+        let sig = Signature { sigma, v };
+
         Federation::new(
             TEST_KEYS.pubkeys()[0],
-            0,
+            10,
             Some(3),
             Some(node_vss(0)),
-            TEST_KEYS.aggregated(),
+            XField::AggregatePublicKey(TEST_KEYS.aggregated()),
+            None,
+            Some(sig),
         )
     }
 
+    fn valid_federation_maxblocksize() -> Federation {
+        let mut federation = valid_federation();
+
+        federation.xfield = XField::MaxBlockSize(400000);
```

Comment:
Are there any restrictions on the maximum or minimum MaxBlockSize?
If a small value (maybe 170bytes) is specified, will TapyrusCore reject the block?

According to https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/signedblocks.md, the minimum Block Header is 170byte.

Reply:
> Are there any restrictions on the maximum or minimum MaxBlockSize?

Tapyrus Core requires the minimum block size to be 1000 bytes. There is no maximum limit other than the 4 byte capacity allowed.

>If a small value (maybe 170bytes) is specified, will TapyrusCore reject the block?

Yes, Tapyrus Core does not allow max block size in xfield to be set below 1000 bytes. Also, BlockAssembler in Tapyrus Core uses the same minimum limit. However if a smaller size valid block is created outside of Tapyrus Core like using tools like our test framework, there is no size restriction. The block is accepted by the node after validation.